cmake-ide.el

```diff
@@ -316,9 +324,11 @@ This works by calling cmake in a temporary directory (or cmake-ide-build-dir)
 (defun cmake-ide--register-callback ()
   "Register callback for when CMake finishes running."
   (set-process-sentinel (get-process "cmake")
-                        (lambda (_process _event)
+                        (lambda (process _event)
```

Comment:
?

Reply:
Because I use process parameter to check process exit status, I remove the underscore (if I understand well, underscore in front of parameter tells the parameter is not used...)

Reply:
Ah, I must have missed that, sorry.

Reply:
I was not sure, now I am : 
https://www.gnu.org/software/emacs/manual/html_node/elisp/Using-Lexical-Binding.html
"(To silence byte-compiler warnings about unused variables, just use a variable name that start with an underscore. The byte-compiler interprets this as an indication that this is a variable known not to be used.)"